.travis.yml

```diff
@@ -7,9 +7,9 @@ install:
 - GO111MODULE=off go get github.com/mattn/goveralls
 - GO111MODULE=off go get golang.org/x/tools/cmd/goimports
 go:
-- "1.11"
-- "1.12"
-- "1.13"
+- "1.13.11"
+- "1.14.7"
```

Comment:
I think for simplicity, let's just set this to 1.14 and 1.13.

Reply:
@jtopjian fixed.